conf/distro/iot2050-debian-qemu.conf

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) Siemens AG, 2023
+#
+# Authors:
+#  Enes �olpan <enes.colpan@siemens.com>
```

Comment:
Is this UTF8 in your name and just a limitation of github in the visualization?

Reply:
I guess this is a typo -- see recipes-core/images/iot2050-image-qemu.bb

Reply:
File is deleted, no problem anymore :)